src/dispensary/index.js

```diff
@@ -0,0 +1,39 @@
+import HASHES from 'dispensary/hashes.txt';
```

Comment:
This import means that the hashes.txt data will be now bundled in the addons-linter repo.

At the moment I think this is fine (even if previously it was likely part of the dispensary bundle and not of the addons-linter bundle), and maybe even good (one less file that the linter would need to access).

And so I think we should just add an inline comment right above `import HASHES ...` just to point this out (e.g. something like `// The entire content of hashes.txt will be part of the addons-linter.js bundle`).

The hashes.txt file is currently ~100kB, and [dist/addons-linter.js bundle file on addons-linter v4.0.0 is 712kB](https://unpkg.com/browse/addons-linter@4.0.0/dist/), that means that I expect the `dist/addons-linter.js` bundle to become ~800kB with this changes, which seems still more than reasonable increase to be ok with.

@willdurand how that sounds? (basically "adding a note in the README.md section and an inline comment right above this line" would be the only changes I'm requresting)

Reply:
Sounds good, useful comments are always good.

I am fine with the increase of the bundle size. Ultimately, the hashes.txt content is loaded from a JS file eventually. The fact that it is going to be from the addons-linter bundle and not the dispensary bundle is an implementation detail. The same `txt` file was embedded in a bundle before.

Reply:
@willdurand :+1: yep, I went a bit more into the detail (just for the sake of leaving a record of what we have evaluated in practice), but ultimately that is exactly what I meant.